core/pkg/eval/json_evaluator.go

```diff
@@ -122,6 +122,11 @@ func (je *JSONEvaluator) ResolveAllValues(ctx context.Context, reqID string, con
 	var err error
 	allFlags := je.store.GetAll()
 	for flagKey, flag := range allFlags {
+		if flag.State == Disabled {
+			// ignore evaluation of disabled flag
+			continue
```

Comment:
Think we can add a simple test case for this?

Reply:
I added a validation to the exisiting test in the latest commit [1]. 

[1] - https://github.com/open-feature/flagd/pull/672/commits/eef88dbc2bd8dbb5aaf0d2362fa5711a05dc9151

Reply:
perfect!